experimental/FTheoryTools/src/TateModels/attributes.jl

```diff
@@ -114,7 +114,7 @@ Normal toric variety without torusfactor
 ```
 """
 @attr AbstractNormalToricVariety function toric_base_space(t::GlobalTateModel)
-    base_fully_specified(t) || @info("Base space was not fully specified. Returning AUXILIARY base space.")
+    base_fully_specified(t) || @vprint :GlobalTateModel 1 "Base space was not fully specified. Returning AUXILIARY base space."
```

Comment:
I think you may also need to use `add_verbosity_scope(:GlobalTateModel)` as described in the `@vprint` docstring?

Reply:
Got it. Thank you!